src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md

```diff
@@ -50,43 +56,42 @@ Adding a component for author in crosswalk site is like adding any other custom
 	  }
 ]
 ```
-### Component Model in `component-models.json to leverage Standard Edge Delivery`
+- `id`: can be any value. In the example above it is called `custom-asset-one`.
```

Comment:
@gazal2708, does the `id` correspond to the "block in which the Asset Picker should be used for adding Images"?

Can you suggest a better description compared to what we have above so that the role of `id` (and the default) is clearer to the developer reading the docs?

Reply:
taking it to https://github.com/AdobeDocs/uix/pull/139